src/AST-Core/RBMethodNode.class.st

```diff
@@ -708,7 +708,7 @@ RBMethodNode >> statements: aCollection [
 
 { #category : #accessing }
 RBMethodNode >> stop [
-	^ self newSource size
+	^(self sourceCode ifNotNil: [:src | src] ifNil: [ self formattedCode ]) size
```

Comment:
Why not simply `^ (self sourceCode ifNil: [ self formattedCode ]) size`?

Reply:
yes. But I will do that in another PR. I want this in as it destroys my productivity.

Reply:
We should in general not hold up fixes due to trivial things like this. Better do another PR